### ISSUE TITLE
Mined and unsent Tfuel tokens to Binance wallet address

### ISSUE THREAD

**Issue by FBRubinato** [thetatoken/guardian-mainnet-guide#63]
I already have 24 tokens mined through Theta Edge Node, I linked Binance's address to receive the mined TFUEL Tokens, and so far the tokens have not been sent there.

Can anybody help me ?

**Comment by Mendez849** [comment]
Hello Mate @FBRubinato , Sorry to hear about your issue, you can contact Theta Network Support on Telegram via the weblink www.t.me/thethetanetwork. They'll be happy to resolve your issue. Cheers.

**Comment by clintbugsdev** [comment]
Any progress? I also used Binance's TFuel address and still waiting for the token to be received. TIA

**Comment by FBRubinato** [comment]
Yes, I received the tokens in the first week of the following month in the Binance wallet


________________________________
De: Clinton Bugtong ***@***.***>
Enviado: Monday, April 5, 2021 10:53:31 PM
Para: thetatoken/guardian-mainnet-guide ***@***.***>
Cc: FBRubinato ***@***.***>; Mention ***@***.***>
Assunto: Re: [thetatoken/guardian-mainnet-guide] Mined and unsent Tfuel tokens to Binance wallet address (#63)


Any progress? I also used Binance's TFuel address and still waiting for the token to be received. TIA

**Comment by clintbugsdev** [comment]
Is that mean that your mined tokens from February was just sent to you this April? Please correct me if I am wrong. By the way, my tokens was already sent to Binance but still waiting for confirmation. How many hours or days you had to wait to be actually received it? Thanks again.

**Comment by FBRubinato** [comment]
The tokens are sent the following month, if you mined in March, you will receive it in April. When you go to Binance, mine usually take 1 day to confirm the transaction!


________________________________
De: Clinton Bugtong ***@***.***>
Enviado: Tuesday, April 6, 2021 5:56:48 AM
Para: thetatoken/guardian-mainnet-guide ***@***.***>
Cc: FBRubinato ***@***.***>; Mention ***@***.***>
Assunto: Re: [thetatoken/guardian-mainnet-guide] Mined and unsent Tfuel tokens to Binance wallet address (#63)


Is that mean that your mined tokens from February was just sent to you this April? Please correct me if I am wrong. By the way, my tokens was already sent to Binance but still waiting for confirmation. How many hours or days you had to wait to be actually received it? Thanks again.

**Comment by clintbugsdev** [comment]
Just got it. LOL

**Comment by FBRubinato** [comment]
Good


________________________________
De: Clinton Bugtong ***@***.***>
Enviado: Tuesday, April 6, 2021 1:14:14 PM
Para: thetatoken/guardian-mainnet-guide ***@***.***>
Cc: FBRubinato ***@***.***>; Mention ***@***.***>
Assunto: Re: [thetatoken/guardian-mainnet-guide] Mined and unsent Tfuel tokens to Binance wallet address (#63)


Just got it. LOL
